### PR TITLE
all jobs: additional checker for exporting plugin_version variable  (for released versions)

### PR DESCRIPTION
**Pull request by naumvd95**
contrail: small bugfix

### CODE REVIEW

File: servers/partner-ci/builders/nsxt.test.sh [hunk 1]
```diff
@@ -25,9 +25,9 @@ fi
 [[ $SNAPSHOTS_ID == *"lastSuccessfulBuild"* ]] && \
   export SNAPSHOTS_ID=$(grep -Po '#\K[^ ]+' < snapshots.params)
 
-
+if [ "${SNAPSHOTS_ID}" != "released" ]; then
   export PLUGIN_VERSION=$(grep "PLUGIN_VERSION" < build.plugin_version | cut -d= -f2 )
```
Comment (rkhozinov) [inline servers/partner-ci/builders/nsxt.test.sh]: should be the same as in the file above

File: servers/partner-ci/builders/nsxv.test.sh [hunk 2]
```diff
@@ -25,9 +25,9 @@ fi
 [[ $SNAPSHOTS_ID == *"lastSuccessfulBuild"* ]] && \
   export SNAPSHOTS_ID=$(grep -Po '#\K[^ ]+' < snapshots.params)
 
-
-export PLUGIN_VERSION=$(grep "PLUGIN_VERSION" < build.plugin_version | cut -d= -f2 )
-
+if [ "${SNAPSHOTS_ID}" != "released" ]; then
```
Comment (rkhozinov) [inline servers/partner-ci/builders/nsxv.test.sh]: the same issue

File: servers/partner-ci/builders/nsxt.test.sh (hunk 1 above)
Comment (naumvd95) [reply servers/partner-ci/builders/nsxt.test.sh]: what file?

Comment (rkhozinov) [reply servers/partner-ci/builders/nsxt.test.sh]: `servers/partner-ci/builders/manila.test.sh` for example

**Review by rkhozinov — APPROVED** [review]

**Pull request merged**